src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -576,6 +576,13 @@ def process_repair(manager, broker, dry_run):
     show_default=True,
     help='Include extras in the `.aiida_node_metadata.yaml` written for every `ProcessNode`.',
 )
+@click.option(
+    '--also-remote',
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help='If true, try to obtain also intermediate files on the remote computer that were not initially retrieved.',
```

Comment:
Perhaps give a bit more detail here, e.g., mention that this requires opening a connection to the remote computer and that it possible cannot retrieve the files if the remote folder has been altered or deleted entirely in the meantime